src/v2/Apps/Artist/Components/ArtistHeader.tsx

```diff
@@ -105,100 +80,28 @@ export class ArtistHeader extends Component<Props> {
   }
 }
 
-@track()
 export class LargeArtistHeader extends Component<Props> {
-  @carouselSlideTrack((_props, _state, [slide]) => {
-    return {
-      action_type: Schema.ActionType.Click,
-      // TODO: Or keep using ‘thumbnail’ as per old Force schema
-      subject: "carouselSlide",
-      // TODO: Are you sure this is no longer needed? Like, do we not need to
-      //       identify the specific slide?
-      destination_path: slide.href,
-    }
-  })
-  onClickSlide(slide) {
```

Comment:
To avoid losing tracking continuity, I would copy this tracking call down in the new carousel that you added.